content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_print.md

```diff
@@ -17,7 +17,7 @@ Print configuration
 
 
 This command prints configurations for subcommands provided.
-For details, see: https://godoc.org/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta2
+For details, see: https://godoc.org/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta3
```

Comment:
Hi @neolit123 .
Some of the files in this PR are auto-generated.
Do you expect the changes to be generated from the source code?

Reply:
that was an oversight. the source code should produce the "v1beta3" change.
updated.